mdal/frmts/mdal_ugrid.cpp

```diff
@@ -878,12 +1091,6 @@ void MDAL::DriverUgrid::writeVariables( MDAL::Mesh *mesh )
       faceIndex += facesRead;
     }
   }
-
-  // Time values (not implemented)
-  mNcFile->putDataDouble( timeId, 0, 0.0 );
-
-  // Turning on define mode
-  nc_redef( mNcFile->handle() );
```

Comment:
why this is removed?

Reply:
for times, same reason as above.

For define mode, I think it is an old left over. Define mode has to be set before adding new variables, not at the end.